packages/components/src/components/radio/radio.stories.ts

```diff
@@ -18,106 +24,48 @@ export default {
  */
 
 export const Default = {
+  name: 'Default',
   render: (args: any) => {
     return generateTemplate({ args });
   }
 };
 
 /**
- * Use the disabled attribute to disable a input radio. Clicks will be suppressed until the disabled state is removed
- */
-
-export const DisabledAndSize = {
-  name: 'Disabled × Size',
-  parameters: { controls: { exclude: ['disabled', 'size', 'default'] } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        x: {
-          type: 'attribute',
-          name: 'disabled',
-          values: [false, true]
-        },
-        y: {
-          type: 'attribute',
-          name: 'size',
-          values: ['lg', 'sm']
-        }
-      },
-      constants: { type: 'attribute', name: 'disabled', value: true },
-      args
-    });
-  }
-};
-
-/**
- * Use the `size` attribute to change the size of the input radio. This attribute affects the font-size within the element, while the element itself remains the same size.
+ * Use the `size` attribute to change the size of the input radio. This attribute affects the font-size within the element, while the element itself remains the same size. The default is `lg`. In tight spaces, consider using `sm`.
  */
 
 export const Size = {
-  parameters: { controls: { exclude: ['size'] } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        x: { type: 'attribute', name: 'size' }
-      },
-      args
-    });
-  }
+  name: 'Size',
+  render: () => html`
+    <div class="flex gap-12">
+      <sd-radio value="1" size="lg">Large</sd-radio>
+      <sd-radio value="2" size="sm">Small</sd-radio>
+    </div>
+  `
 };
 
 /**
- * Use the `invalid` attribute to mark the element is not in a format or a value the application will accept.
+ * Use the `invalid` attribute to mark that the element is not in a format or a value the application will accept.
```

Comment:
Delete  * Use the `invalid` attribute to mark that the element is not in a format or a value the application will accept.


Add:
For an invalid radio-group (since radios always come in groups) an error-text underneath the group is mandatory.

Reply:
Show Invalid as last section on this page.

Reply:
I can add this message but we should keep some sort of description for the `disabled` attribute. The new text does not make any reference to it. What do you think?

Reply:
We should keep some sort of reference to the use of the `invalid` attribute. With the new description there is no mention to it. What do you think?

Reply:
@coraliefeil Is the first part 'Figma-only'? If not, why should we delete it? In Figma, we have both descriptions without any indication if they are 'Figma-only' or not.

Reply:
I think it makes sense to have a mention of how the invalid attribute works from a technical perspective. I just pushed a change that attempts to consider both perspectives. Let me know if this works :)

Reply:
Comments above are probably misleading ... in our worksession we agree on this text for Invalid:
The invalid attribute marks the radio as invalid. For an invalid radio-group (since radios always come in groups) an error-text underneath the group is mandatory.

The text is in sync.

From my POV all is good and approved.